subgraphs/makerdao/schema.graphql

```diff
@@ -460,7 +460,7 @@ type Market @entity {
   liquidationPenalty: BigDecimal!
 
   " Token that need to be deposited to take a position in protocol "
-  inputToken: Token!
+  inputToken: Token
```

Comment:
This change breaks the schema. Why do you need this?

Reply:
i changed it back